### ISSUE TITLE
Keyboard navigation and mouse hover styling is wonky

### ISSUE THREAD

**Issue by colinbdclark** [colinbdclark/bubbles#20]
There are several interrelated issues related to keyboard navigation and mouse styling that need to be fixed:

- When the Composition Panel is hidden, it's contents continue to be selectable and activatable because the panel is hidden with <code>opacity: 0;</code>. The contents of the panel (though not the panel itself, because its show/hide button needs to remain visible) should be set to <code>display: none;</code> so that they are taken out of the tab order.
- The add video button no longer shows focusing and hover styling to indicate that it can be activated
- The video's border no longer show a highlight when the add button is selected or hovered over
- The add layer button no longer shows hover styling
